plugins/github.js

```diff
@@ -83,6 +83,58 @@ function pullRequestClosed(options, request, reply) {
 }
 
 /**
+ * Stop any running builds and start the build for the synchronized pull-request
+ * @method pullRequestClosed
+ * @param  {Object}       options
+ * @param  {String}       options.eventId    Unique ID for this GitHub event
+ * @param  {String}       options.jobId      Identifier for the Job
+ * @param  {String}       options.name       Name of the job (PR-1)
+ * @param  {Hapi.request} request Request from user
+ * @param  {Hapi.reply}   reply   Reply to user
+ */
+function pullRequestSync(options, request, reply) {
+    const eventId = options.eventId;
+    const jobId = options.jobId;
+    const name = options.name;
+
+    async.waterfall([
+        // Lookup all the builds for the jobId
+        async.apply(Build.getBuildsByJobId.bind(Build), { jobId }),
```

Comment:
`getBuildsForJobId` is the method name:
https://github.com/screwdriver-cd/models/blob/master/lib/build.js#L152